modules/claims_api/spec/requests/v0/disability_compensation_request_spec.rb

```diff
@@ -45,15 +45,6 @@
       end
     end
 
-    it 'should return an unsuccessful response with an error message' do
-      VCR.use_cassette('evss/intent_to_file/active_compensation') do
-        post path, params: data, headers: headers
-        parsed = JSON.parse(response.body)
-        expect(response.status).to eq(422)
-        expect(parsed['errors'].first['details']).to eq('Intent to File Expiration Date not valid, resubmit ITF.')
-      end
-    end
```

Comment:
Seems like removing the ITF validation check would just alter the expected behavior which would entail updating this expectation as opposed to deleting the example altogether... or is this VCR cassette no longer valid?

Reply:
I would say in this case the VCR is no longer necessary.

Reply:
The vcr is still used in other tests. This test has been removed but the current response is covered in other tests since we no longer throw this 422

Reply:
the places where the cassette was used but not needed have been removed.